packages/moonstone/TooltipDecorator/TooltipDecorator.js

```diff
@@ -163,6 +164,9 @@ const TooltipDecorator = hoc(defaultConfig, (config, Wrapped) => {
 			super(props);
 
 			this.TOOLTIP_HEIGHT = ri.scale(18); // distance between client and tooltip's label
+			/* global MutationObserver ResizeObserver */
+			this.mutationObserver = new MutationObserver(this.setTooltipLayoutJob.start);
+			this.resizeObserver = new ResizeObserver(this.setTooltipLayoutJob.start);
```

Comment:
In the interest of cross-platform support, I would like to only conditionally create a `ResizeObserver` since it is relatively new and isn't widely supported. For example, it will throw an error in Firefox and Safari. Realize it means that we won't have the same positioning resilience as in chromium but that's acceptable for now. We can revisit alternatives in the future.